### ISSUE TITLE
Score in Table 1

### ISSUE THREAD

**Issue by byeongkeun-kang** [bioinf-jku/TTUR#11]
I have trained "GoodGenerator" (in the provided code) with TTUR (using the given default hyperparameters) for a couple of times. My trained model achieves about 29 FID score for the CIFAR-10 dataset. To achieve the score in Table 1, should I pick the best score among multiple training and multiple generations? Or, should I change the hyperparameters? Thank you.

**Comment by mhex** [comment]
Hi Byeongkeun, yes pick the best FID among all training runs and epochs. We can do that because we don't have a test set here. Greets Martin

**Comment by byeongkeun-kang** [comment]
Could you share your trained model that performs the best? Thanks.

**Comment by mhex** [comment]
I'm sorry we can't send you the best model.
